README.md

```diff
@@ -1,65 +1,48 @@
 # gRPC-Go
 
 [![Build Status](https://travis-ci.org/grpc/grpc-go.svg)](https://travis-ci.org/grpc/grpc-go)
-[![GoDoc](https://godoc.org/google.golang.org/grpc?status.svg)](https://godoc.org/google.golang.org/grpc)
+[![GoDoc](https://godoc.org/google.golang.org/grpc?status.svg)][API]
 [![GoReportCard](https://goreportcard.com/badge/grpc/grpc-go)](https://goreportcard.com/report/github.com/grpc/grpc-go)
 
-The Go implementation of [gRPC](https://grpc.io/): A high performance, open
-source, general RPC framework that puts mobile and HTTP/2 first. For more
-information see the [gRPC Quick Start:
-Go](https://grpc.io/docs/languages/go/quickstart/) guide.
+The Go implementation of [gRPC][]: A high performance, open source, general RPC
+framework that puts mobile and HTTP/2 first. For more information see the [Go
+gRPC docs][], or jump directly into the [quick start][].
 
-Installation
-------------
+## Prerequisites
 
-To install this package, you need to install Go and setup your Go workspace on
-your computer. The simplest way to install the library is to run:
+- **[Go][]**, any one of the **three latest major** [releases of Go][].
 
-```
+## Installation
+
+The simplest way to install the grpc-go package is to run the following command:
+
+```console
 $ go get -u google.golang.org/grpc
 ```
 
-With Go module support (Go 1.11+), simply `import "google.golang.org/grpc"` in
-your source code and `go [build|run|test]` will automatically download the
-necessary dependencies ([Go modules
-ref](https://github.com/golang/go/wiki/Modules)).
+With [Go module][] support (Go 1.11+), simply `import "google.golang.org/grpc"`
+in your source code and `go [build|run|test]` will automatically download the
+necessary dependencies.
 
-If you are trying to access grpc-go from within China, please see the
+> **Note:** If you are trying to access grpc-go from within **China**, see the
 [FAQ](#FAQ) below.
 
-Prerequisites
--------------
-gRPC-Go officially supports the
-[three latest major releases of Go](https://golang.org/doc/devel/release.html).
+## Learn more
 
-Documentation
```

Comment:
The "Documentation" contained more than doc links, so I renamed it to "Learn more", and folded the "Performance" link into the list.